subworkflows/local/makeContaminantIndex/main.nf

```diff
@@ -17,24 +18,30 @@ include { BOWTIE2_INDEX } from "../../../modules/local/bowtie2"
 
 workflow MAKE_CONTAMINANT_INDEX {
     take:
-        cow_url
-        pig_url
-        mouse_url
-        carp_url
-        ecoli_url
+        genome_urls
```

Comment:
Format this as a list/tuple in the config file and update parsing accordingly.

Reply:
Done.